templates/taxi/car_detail.html

```diff
@@ -2,6 +2,10 @@
 
 {% block content %}
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+   <div>
+      <a  class="btn btn-primary" href="{% url 'taxi:car-update' pk=car.id%}">Update</a>
+      <a class="btn btn-primary" href="{% url 'taxi:car-delete' pk=car.id%}">Delete</a>
+  </div>
   <h1>Drivers</h1>
```

Comment:
maybe using of another tag would be better? `h2` or `h3` for example? I don't like an interface of this page